swift_scality_backend/utils.py

```diff
@@ -68,8 +69,11 @@ def wrapped(*args, **kwargs):
         # Get & bump call identifier, assume non-preemptive threading
         ctid, tid[0] = tid[0], tid[0] + 1
 
-        all_args = inspect.getcallargs(f, *args, **kwargs)
-        logger.debug('==> %s (%d): call %r', name, ctid, all_args)
+        if sys.version_info[0:2] > (2, 6):
```

Comment:
No need for the `[0:2]` slice if changed into `if sys.version_info >= (2, 7)` (which IMHO captures the idea better).